core/services/chainlink/application.go

```diff
@@ -289,26 +294,37 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	} else {
 		globalLogger.Debug("Off-chain reporting disabled")
 	}
+	terraORM := terra.NewORM(db)
 	if cfg.FeatureOffchainReporting2() {
 		globalLogger.Debug("Off-chain reporting v2 enabled")
 		// TODO: parameterize poll period
-		// TODO: pull client config from nodes
-		// TODO: If terra enabled
-		chainID := "42"
-		tc, err := terraclient.NewClient("42", "0.01", "1.5", "TODO", "TODO", 10, globalLogger)
+		nodes, _, err := terraORM.Nodes(0, 1)
```

Comment:
Temporary caveat: we are initializing the tx manager on startup, so after adding a node via remote CLI command, a restart is required to use the node.

Reply:
This applies to all chain/node changes currently, there is a ticket in the backlog to make it do hot loading https://app.shortcut.com/chainlinklabs/story/24556/need-to-be-able-to-boot-with-no-eth-chains-eth-nodes-and-add-them-later-and-have-everything-spin-up-as-expected